homeassistant/components/isy994/helpers.py

```diff
@@ -428,7 +428,7 @@ def convert_isy_value_to_hass(
     if uom in (UOM_DOUBLE_TEMP, UOM_ISYV4_DEGREES):
         return round(float(value) / 2.0, 1)
     if precision not in ("0", 0):
-        return int(round(float(value) / 10 ** int(precision), int(precision)))
+        return float(round(float(value) / 10 ** int(precision), int(precision)))
```

Comment:
Isn't it already a float returned by round? We have a guard against 0 decimals which would have made it an integer.

Reply:
If mypy doesn't know, I'd cast it.
```suggestion
        return cast(float, round(float(value) / 10 ** int(precision), int(precision)))
```

Reply:
`cast` is cleaner for sure 👍